packages/custom_lint/test/src/workspace_test.dart

```diff
@@ -261,10 +279,16 @@ Future<Directory> createWorkspace(
   return dir;
 }
 
-Directory createTemporaryDirectory() {
-  final dir = Directory.current //
-      .dir('.dart_tool')
-      .createTempSync('custom_lint_test');
+Directory createTemporaryDirectory({bool local = false}) {
```

Comment:
I don't understand what this flag is about. But I'd be happy to have `local` always on.

Reply:
Since the project directory resolution logic looks up from the directory to find the closest enclosing project directory in order to test the failure case we need it to be outside the custom_lint package

Reply:
It would be best for local to be off by default, so that we ensure the tests don't pick up the pubspec.yaml of custom_lint.